src/main/java/com/knappsack/swagger4springweb/util/AnnotationUtils.java

```diff
@@ -95,6 +101,35 @@ public static String getMethodRequestMappingValue(Method method) {
         return annotatedParameters;
     }
 
+    /**
+     * @param method
+     *            Method
+     * @return List<Parameter> - if the method is annotated with ApiImplicitParams that contain parameters definitions
+     */
+    public static List<Parameter> getImplicitParameters(Method method) {
+        final List<Parameter> parameters = Lists.newArrayList();
+        final ApiImplicitParams apiImplicitParams = findApiImplicitParams(
+                Lists.newArrayList(method.getDeclaredAnnotations()));
+        if (apiImplicitParams != null) {
+            for (final ApiImplicitParam apiImplicitParam : apiImplicitParams.value()) {
+                parameters.add(new Parameter(apiImplicitParam.name(), Option.apply(apiImplicitParam.value()),
+                        Option.apply(apiImplicitParam.defaultValue()),
+                        apiImplicitParam.required(), apiImplicitParam.allowMultiple(),
+                        apiImplicitParam.dataType().toLowerCase(), getAllowableValues(apiImplicitParam),
```

Comment:
Can we have NPE on `dataType()`? We have default value, but if it's explicitly set to null?